example/credentials.example.yaml

```diff
@@ -1,3 +1,4 @@
 app-id: <Your app ID>
 app-key: <Your app key>
-app-secret: <Your app secret>
\ No newline at end of file
+app-secret: <Your app secret>
+app-cluster: <Your app cluster. Omit for default>
```

Comment:
One small point:  I don't think we should be advocating omitting the cluster. I think it makes it less confusing if the cluster is *always* specified in all scenarios.